### ISSUE TITLE
Add term posterior notching vertebral bodies

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10376]
**Preferred term label:**
Posterior notching vertebral bodies

**Synonyms**

**Definition (free text, please give PubMed ID)**
Exaggerated posterior vertebral notches
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/001c7591-5a39-4f99-978b-ba21060a0f0d)
From PMID: 12624140

**Parent term (use hpo.jax.org/app)**
Abnormal form of the vertebral bodies HP:0003312

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000777
- label: Posterior vertebral body notching
- parents: [Abnormal form of the vertebral bodies (HP:0003312)](https://hpo.jax.org/app/browse/term/HP:0003312)
- pmid: PMID:12624140
- def: Exaggerated posterior vertebral notches.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10376
- synonyms:
